src/app/containers/Include/amp/VjAmp.jsx

```diff
@@ -18,31 +28,98 @@ const IncludeGrid = styled(GridItemConstrainedMedium)`
   display: grid;
 `;
 
+const AmpIframe = ({ children, className, width, height, src }) => (
+  <amp-iframe
+    class={className}
+    width={width}
+    height={height}
+    layout="responsive"
+    sandbox="allow-scripts allow-same-origin allow-top-navigation-by-user-activation allow-forms"
+    resizable
+    src={src}
+  >
+    {children}
+  </amp-iframe>
+);
+
+const StyledAmpIframe = styled(AmpIframe)`
+  > [overflow] {
+    background: linear-gradient(
+      0deg,
+      #fff 0%,
+      rgba(255, 255, 255, 1) 75%,
+      rgba(255, 255, 255, 0) 100%
+    );
+    display: flex;
+    justify-content: center;
+
+    &::after {
+      background-color: #fff;
+      border-top: ${KEYLINE_WIDTH} solid ${C_EBON};
+      content: '';
+      display: block;
+      height: 50%;
+      left: 0;
+      position: absolute;
+      top: 50%;
+      width: 100%;
+      z-index: -10;
+    }
+
+    button {
+      ${({ service }) => getSansBold(service)}
+      ${GEL_PICA}
+      background-color: ${C_EBON};
+      border: none;
+      color: #fff;
+      cursor: pointer;
+      display: block;
+      padding: ${GEL_SPACING} ${GEL_SPACING_DBL};
+      &:hover {
+        text-decoration: underline;
+      }
+    }
+  }
+`;
+
 const VjAmp = ({ ampMetadata: { imageWidth, imageHeight, image, src } }) => {
+  const { service } = useContext(ServiceContext);
   return (
     <>
       <AmpHead />
       <IncludeGrid>
-        <amp-iframe
+        <StyledAmpIframe
           width={imageWidth}
           height={imageHeight}
-          layout="responsive"
-          sandbox="allow-scripts allow-same-origin allow-top-navigation-by-user-activation allow-forms"
           src={src}
+          service={service}
         >
+          <div overflow="true">
+            <button type="button" aria-label="Show more">
```

Comment:
Please remove the aria-label, this is repeating the text inside the element...

Reply:
Unless we are actually going to say what we are going to show more of? Which we should... Do we have some functionality that ensures the content that is not shown is hidden from AT until this button is clicked? I'm not sure it's a very accessibly implementation we currenlty have on live...

Reply:
Don't forget this is AMP and we're bound to their specification: https://amp.dev/documentation/components/amp-iframe/#iframe-resizing. We can probably remove the aria-label here now that this is a button and not a div though.